.github/workflows/deploy-docs.yml

```diff
@@ -15,12 +15,12 @@ jobs:
 
     steps:
     - name: checkout
-      uses: actions/checkout@v4
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  # v4.2.2
```

Comment:
Out of curiosity, why is this hash specification needed now, instead a simpler version number?

Reply:
This is a security recommendation from GitHub. Version tags are editable, hashes aren't. The comment version does get updated with dependabot, so we won't lose track of them when merge the dependabot updates.